pkg/config/static/entrypoints.go

```diff
@@ -163,6 +164,13 @@ func (u *UDPConfig) SetDefaults() {
 	u.Timeout = ptypes.Duration(DefaultUDPTimeout)
 }
 
+type TSNetConfig struct {
+	Hostname  string `description:"Hostname for the node; if empty, defaults to the endpoint's name" json:"hostname,omitempty" toml:"hostname,omitempty" yaml:"hostname,omitempty" export:"true"`
+	Ephemeral bool   `description:"Register the node as ephemeral." json:"ephemeral,omitempty" toml:"ephemeral,omitempty" yaml:"ephemeral,omitempty" export:"true"`
+	AuthKey   string `description:"Auth key for the node (overrides the TS_AUTHKEY env var)." json:"authKey,omitempty" toml:"authKey,omitempty" yaml:"authKey,omitempty" export:"true"`
+	// TODO: Funnel to enable listening on a funnel
```

Comment:
Maybe also add `ControlURL` to be able to run integration tests against a headscale instance in the future?

Reply:
Done